jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/auth/Auth.java

```diff
@@ -103,4 +104,20 @@ public static boolean allow(String user, AuthPolicy policy, Runnable notAllowed)
         notAllowed.run();
         return false;
     }
+
+    /**
+     * Calculate the value of the "Authentication" HTTP header for basic auth. Basic
+     * auth is not secure when used over HTTP (the password can be extracted). Use
+     * with HTTPS is better.
+     * <p>
+     * Unlike digest auth, basic auth can be setup without an extra round trip to the
+     * server, making it easier for scripts where teh body is not replayable.
```

Comment:
s/teh/the (I think Java 8+ javadoc is stricter and requires the closing `</p>`, unless we are passing those args to reduce strictness during build)

Reply:
Done

Reply:
Thanks for raising this. I can't find a definitive answer by searching the web.

Not sure about the `<p>`. It is legal HTML (not xHTML). I don't get a warning/error in Eclipse. The POM has `<doclint>none</doclint>` and given the scale and age of Jena, other places will have no closing `</p>`.

Elsewhere, I have projects using Java11 and no special configuration of the javadoc plugin.